src/community/authkey/src/main/java/org/geoserver/security/AuthenticationKeyFilterConfig.java

```diff
@@ -75,4 +81,39 @@ public void setUserGroupServiceName(String userGroupServiceName) {
     public void setMapperParameters(Map<String, String> mapperParameters) {
         this.mapperParameters = mapperParameters;
     }
+
+    public void setAllowEnvParametrization(boolean allowEnvParametrization) {
```

Comment:
What is going to set this property? I thought env parametrization was controlled by a single system variable, "-DALLOW_ENV_PARAMETRIZATION=true", see https://docs.geoserver.org/stable/en/user/community/backuprestore/configtemplate.html

Code wise it seems everything is referring to GeoServerEnvironment.ALLOW_ENV_PARAMETRIZATION.

Reply:
used by the tests